### PR TITLE
Update to Node.js 20.x

### PR DESCRIPTION
**Pull request by junlarsen**
Node 18 is in maintenance mode, while 20 is in active development. We should use 20.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
       - name: Setup Node.js environment
         uses: actions/setup-node@v3
         with:
-          node-version: 18.18.0
+          node-version: 20.12.2
```
Comment (henrikskog) [inline .github/workflows/ci.yml:59]: Why not 20.12.7?

File: package.json [hunk 2]
```diff
@@ -35,16 +35,13 @@
     "apps/*"
   ],
   "engines": {
-    "node": ">=18.18.0",
-    "pnpm": ">=8.7.6"
+    "node": ">=20.12.2",
```
Comment (henrikskog) [inline package.json:38]: Why kot 20.12.7?

**Review by henrikskog — APPROVED** [review]
Nice! Got two questions, but looks good!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (junlarsen) [reply .github/workflows/ci.yml:59]: probably typod lol

**Comment by junlarsen** [comment]
Vercel doesn't support pnpm 9 yet

File: .github/workflows/ci.yml (hunk 1 above)
Comment (junlarsen) [reply .github/workflows/ci.yml:59]: node 20.12.7 isn't a thing

**Pull request merged**